### ISSUE TITLE
InputMask: p:ajax without event=blur doesn't work

### ISSUE THREAD

**Issue by erickdeoliveiraleal** [primefaces/primefaces#6161]
**Describe the defect**
In 8.0 if you use inputMask with p:ajax with no events it worked, master 9.0 does'nt

**Reproducer**
https://github.com/erickdeoliveiraleal/primefaces-test/tree/input

**Expected behavior**
It worked without event event="blur"

**Example XHTML**
```html
<h:form>
		<p:inputMask mask="99[9]9" type="tel" styleClass="Fs20" 
			style="text-transform:uppercase;text-align:center;margin-left:5px"
			size="4" maxlength="4" id="fimplaca" autocomplete="false">
			<p:ajax 
				 process="@this"
				update="@form" listener="#{testView.test()}"
				resetValues="true" />
		</p:inputMask>
	</h:form>
```

**Example Bean**
```java
@Named
@ViewScoped
public class TestView implements Serializable {
    
  
    
      
    public void test() {
       System.out.println(123);
    }

  
}

```

**Comment by tandraschko** [comment]
can you please explain this change?
this was a essential code in older inputnumber versions to support onchange attr, inputnumber copy and ajax.

**Comment by tandraschko** [comment]
good change to provide some integrationtests ;)

**Comment by melloware** [comment]
Sorry I should have explained that the new InputMask does not have its own onchange event so there is no need to do this now as the normal Onchange event works on the Input.  

This code was broken because it was removing the onchange and passing it asa config that the Mask was ignoring.

**Comment by tandraschko** [comment]
ok
its now really the best time to add some integrations tests for it

**Comment by melloware** [comment]
I will add InputMask tests to the Integration Test project.
